man/resolved.conf.xml

```diff
@@ -294,6 +294,12 @@ DNSStubListenerExtra=udp:[2001:db8:0:f102::13]:9953</programlisting>
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestrictInterfaces=</varname></term>
+        <listitem><para>A allow-listed of interface names. When set resolved handles only interfaces which are in allow-list.
```

Comment:
`An allow-listed of interface names. When set, <command>systemd-resolved</command> handles only interfaces which are in the allow-list.`

As usual, please mention that this can be specified multiple times, and the list is cleaned when an empty string is assigned.